src/gui/window_file_list.cpp

```diff
@@ -75,7 +76,8 @@ window_file_list_t::window_file_list_t(window_t *parent, Rect16 rect)
     , color_text(GuiDefaults::ColorText)
     , font(GuiDefaults::Font)
     , padding({ 2, 6, 2, 6 })
-    , ldv(LDV_Get()) {
+    , ldv(LDV_Get())
+    , activeItem(string_view_utf8(), (uint16_t)0) {
     // it is still the same address every time, no harm assigning it again.
```

Comment:
Please remove this comment - it was meant for passing the pointer of LDV to a underlying C-code window. Obsolete now